erpnext_com/www/docs/user/manual/en/setting-up/energy-point-system.md

```diff
@@ -0,0 +1,63 @@
+<!-- add-breadcrumbs -->
+
+### What is Energy Point System?
+Energy Points System is basically a rating/karma system that you can enable for your organization. This system can be used to track performance of a user.
+
+> To enable Energy Point System go to `Energy Point Settings` > check `Enabled`.
+
+
+### How to use Energy Point System?
+To start using this system you need to [create some Energy Point Rules](#how-to-create-energy-point-rules?) so that user can get energy points based on their activities.
+
+### What is Energy Point Rules?
+Energy Point Rules store the information about the activity and the value of points to be alloted.
+
+Energy Point Rule has following fields:
+
+| Field        | Description  |
+| ------------- |:-------------|
+| Reference DocType| Document Type you want to apply rule on eg. Task, ToDo, Issue, etc. |
+| Condition| Condition for the point allocation. <br>eg. `doc.status == 'Closed'`<br>The above condition will check whether the `status` field in the document is 'Closed' and if yes then the points will be alloted to the user.       |
+| Points | Points to be alloted.|
+| User Field | Field from which user will be selected eg. `Resolved By`, `Modified By`, `Owner`, etc. can be used.      |
+| Multiplier Field | Field which stores value for multiplier. This field must have numeric value, this will be multiplied with Points defined in the rule.      |
```

Comment:
This field can take numeric and decimal values. //multiplier can also be 1.5 right? if no, ignore
add: For example 1 = 10 points